tauri-build/src/config.rs

```diff
@@ -84,5 +84,6 @@ fn default_dev_path() -> String {
 }
 
 pub fn get() -> Config {
-  serde_json::from_str(include_str!(concat!(env!("TAURI_DIR"), "/config.json"))).unwrap()
+  serde_json::from_str(include_str!(concat!(env!("TAURI_DIR"), "/config.json")))
```

Comment:
this will change with the refactor to use `tauri.conf.json` - right?

Reply:
I am not entirely sure.  The fixture example config.json is just a single line: `{"devPath": "http://localhost"}`  and then there is the tauri.config.js in there too.